core/session.ts

```diff
@@ -866,6 +868,34 @@ export class Session {
       );
     });
   }
+
+  private getNonUniqueTargets(targets: dataform.ITarget[]): StringifiedSet<dataform.ITarget> {
```

Comment:
this doesn't need to be a method on `Session` - either move it outside of the class or just define it as a function inside `removeNonUniqueActionsFromCompiledGraph`